control/tap-ctl-ipc.c

```diff
@@ -187,6 +187,13 @@ tap_ctl_connect(const char *name, int *sfd)
 
 	memset(&saddr, 0, sizeof(saddr));
 	saddr.sun_family = AF_UNIX;
+
+	if (unlikely(strlen(name) > sizeof(saddr.sun_path) - 1)) {
```

Comment:
This is a random line taken from the linux kernel
```
if (unlikely(!bvl && (gfp_mask & __GFP_DIRECT_RECLAIM)))
```

I think whoever wrote the block layer knows C enough, nevertheless they felt the need to
make it readable.

There is a difference between readable and understanble